arch/cpu/cc26xx-cc13xx/rf-core/ieee-mode.c

```diff
@@ -256,18 +190,13 @@ static uint8_t cmd_ieee_rx_buf[RF_CMD_BUFFER_SIZE] CC_ALIGN(4);
 #define DATA_ENTRY_LENSZ_BYTE 1
 #define DATA_ENTRY_LENSZ_WORD 2 /* 2 bytes */
 
-#define RX_BUF_SIZE 144
+#define RX_BUF_SIZE 142
```

Comment:
This will be a bit problematic if someone were to `#define RX_BUF_INCLUDE_CRC 1`

Reply:
Salivating.  Need this for CC1352.

Reply:
Actually, 142 is sufficient for all config options, including with CRC. Still, point taken - I will change this